Automattic/wp-calypso
# Add Martech ToS user under the key `martechTosUser`.

#### Changes proposed in this Pull Request

This PR adds a new user `martechTosUser` which is used for the [ToS Acceptance Tracking project](pau2Xa-3my-p2).

To use the user in code:

```
	it( 'Log in', async function () {
		const loginPage = new LoginPage( page );
		await loginPage.login( { account: 'martechToSUser' } );
	} );
```

#### Testing instructions

- pull this branch
- decrypt the secrets
- ensure `martechToSUser` exists in the file.

Related to https://github.com/Automattic/wp-calypso/pull/57741.
